install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -50,19 +50,26 @@ schema: |-
     // A subject is an editor, if:
     //  * the user is directly assigned as an editor
     relation editor: user
```

Comment:
I'm surprised we already have this extension in the model. I thought this was gonna be part of the `Fix projects` workstream

Reply:
Yes, I was also surprised. Don't know when it came in. It is not used so far.